devfile.yaml

```diff
@@ -101,8 +100,8 @@ commands:
     exec:
       component: tools
       workingDir: ${PROJECT_SOURCE}
-      commandLine: >-
-        java -jar -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005 target/*.jar
+      commandLine: |
+        mvn spring-boot:run -Drun.jvmArguments='-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005,quiet=y'
```

Comment:
Maybe it's better to remove `run-debug` command at all and use `mvn spring-boot:run -Drun.jvmArguments='-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005,quiet=y'` commandLine in `run-with-mysql`. It's seems more logical for me to have only `run-with-hsqldb` and `run-with-mysql`, both of them will run the application in debug mode.

@vitaliy-guliy  what do you think?

Reply:
Yes, I'm agree with you here and have already reworked `run-with-hsqldb` command and removed `run-debug`.